TSC-Charter.md

```diff
@@ -56,14 +57,29 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC membership shall include (but is not limited to) Maintainers from Core
+Projects as defined in the project lifecycle document and Section 7. The TSC
+may, at its discretion, invite new members to join the TSC who are not
+considered to be Maintainers from Core Projects.
```

Comment:
I agree that it would be better with the simpler statement William suggests. Why say it shall be X but then say unless we want to invite somebody that does not fit X.